src/stylesheets/core/_functions.scss

```diff
@@ -232,22 +234,6 @@ Remove a value from a list
   @return $result;
 }
 
-/*
-----------------------------------------
-strunquote()
-----------------------------------------
-Unquote a string
-----------------------------------------
-*/
-
-@function strunquote($value) {
-  @if type-of($value) == "string" {
-    $value: unquote($value);
-  }
-
-  @return $value;
-}
-
```

Comment:
Per stylelint, this functionality is no longer required or good practice. I've removed it with no ill effects